templates/directus/files/.platform.app.yaml

```diff
@@ -20,7 +20,15 @@ relationships:
 # The hooks that will be triggered when the package is deployed.
 hooks:
     build: |
+        # Use module's example.env file as .env.
         cp node_modules/directus/example.env .env
```

Comment:
Ha!  I literally just wrote a blog post saying to not use `.env` on Platform.sh. :smile: 

Why is it doing this?

Reply:
Totally get you, but seems like this whole thing breaks if a `.env` is not committed, actually. Investigating...

Reply:
Ope, nevermind. Figured it out. Update on the way